src/main/java/com/google/sps/data/GovAdaptor.java

```diff
@@ -18,6 +18,13 @@
  * The adaptors get their information from their respective data sources - the government API is called here. 
  */
 public class GovAdaptor {
+  public static void main(final String[] args) {
+    try {
+      GovAdaptor.extract();
+    } catch (final Exception e) {
+      System.out.println(e.getMessage());
+    }
+  }
```

Comment:
Did you mean to put the main method back in?

Reply:
good catch  - took it out and put it back in for my local dev as discussed but dw it will be out for the production code, this is for the local dev atm